src/shared/tpm2-util.h

```diff
@@ -6,6 +6,12 @@
 #include "json.h"
 #include "macro.h"
 
+typedef enum TPM2Flags {
+        TPM2_FLAGS_USE_PIN = 1,
+        _TPM2_FLAGS_MAX,
```

Comment:
given this is flags field, the _MAX enum doesn't make too much sense, i.e. it would be the OR combination of the lowest and the highest defined bit, but that's not really a maximum value of anything...

Reply:
I agree, removed.